src/main/java/org/folio/service/pieces/flows/delete/PieceDeleteFlowManager.java

```diff
@@ -79,4 +85,26 @@ protected Future<Void> updatePoLine(PieceDeletionHolder holder, RequestContext r
       : pieceDeleteFlowPoLineService.updatePoLine(holder, requestContext);
   }
 
+  public Future<List<Void>> batchDeletePiece (PieceCollection entity, RequestContext requestContext) {
```

Comment:
here is some formatting issue, also we need to accept list of ids to delete instead of full PieceCollection object
```suggestion
  public Future<List<Void>> batchDeletePieces(List<String> pieceIds, RequestContext requestContext) {
```